patch/gfx6/chip/llpcGfx6ConfigBuilder.cpp

```diff
@@ -735,7 +735,7 @@ Result ConfigBuilder::BuildHsRegConfig(
     const auto& pIntfData = m_pContext->GetShaderInterfaceData(shaderStage);
     const auto pResUsage = m_pContext->GetShaderResourceUsage(shaderStage);
     const auto& calcFactor = pResUsage->inOutUsage.tcs.calcFactor;
-    const auto& builtInUsage = pResUsage->builtInUsage.tcs;
+    const auto& tessMode = m_pPipelineState->GetShaderModes()->GetTessellationMode();
```

Comment:
I think my variable name is fine.

Reply:
I suggest this because the modes are for shader. If you define vertexMode, we might not know this is for vertex shader clearly because "vertex" in 3D can denote many items.

Reply:
That is true for vertex. I think it is not so true for the others, especially for tess, as that relates to two shaders (spir-v lets you put some of those modes on either tcs or tes) and it relates to the whole of how tessellation works.

Reply:
Actuall, in GL spec and extension spec, tessellation shader often denote the two stages. I understand why you keep it as "tessellation mode" because GLSL specifies tess qualifier in TES while HLSL specifies it in HS (TCS). It's OK to keep current naming. I just point out this problem when we add vertexMode in the future. Anyway, we make changes when this is encountered.